### DISCUSSION TITLE
Working on Edge in MacOS

### DISCUSSION THREAD

**Discussion by anggoran** [JamesBrill/react-speech-recognition#233] [Q&A]
I tried to use Edge in MacOS and it doesn't work. Is it not supported?
<img width="1217" alt="Screenshot 2024-04-18 at 14 07 11" src="https://github.com/JamesBrill/react-speech-recognition/assets/66579441/013f85d0-e99b-40c4-ba8c-2e7f285fa8f1">

**Comment by Hidesam** [comment]
> I tried to use Edge in MacOS and it doesn't work. Is it not supported? <img alt="Screenshot 2024-04-18 at 14 07 11" width="1217" src="https://private-user-images.githubusercontent.com/66579441/323501156-013f85d0-e99b-40c4-ba8c-2e7f285fa8f1.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3Mzg5MTQ2NDksIm5iZiI6MTczODkxNDM0OSwicGF0aCI6Ii82NjU3OTQ0MS8zMjM1MDExNTYtMDEzZjg1ZDAtZTk5Yi00MGM0LWJhOGMtMmU3ZjI4NWZhOGYxLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNTAyMDclMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjUwMjA3VDA3NDU0OVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTE0YjhmZmY1NTQ3MWI3NWNmY2JkYTgzZjcyNDcyMWVmM2NkNzllYTRlMWVjYmIyZTU2OTZlYjhiMTY2OWNlNWQmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.J06gmvgtfE9zXNWGuYAOK0zB3poxTZtr33bn0YonvHA">

me to，我也是，官方求解决啊

**Comment by anderskristo** [comment]
Same problem here

**Comment by smorimoto** [comment]
Same as https://github.com/JamesBrill/react-speech-recognition/discussions/230
